README.md

```diff
@@ -1,5 +1,6 @@
 # react-native-key-command
```

Comment:
Did we want to add the `@expensify/` prefix later?

Reply:
This is just a README for the repo, so I don't think it's needed?